flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/BuiltInFunctionTestBase.java

```diff
@@ -500,6 +509,11 @@ public static ResultSpec resultSpec(
         return resultSpec(tableApiExpression, sqlExpression, result, dataType, dataType);
     }
 
+    public static ResultSpec resultSpecOnlyContainsSqlExpression(
```

Comment:
There is a `failSQL()` method in `CastFunctionITCase#CastTestSpecBuilder`, if we need to have IT test case for failures.

Reply:
Sorry, ignore this, I was confused.

Reply:
You could simplify the name like `resultSpecSQL` or `sqlResultSpec`.

Reply:
got it.